test/validators/requiredObject.spec.ts

```diff
@@ -1,17 +1,19 @@
-import test from 'ava'
+import { buildValidatorTest } from "../validator-macro.ts";
+import { requiredObject } from "../../src/validators/requiredObject.ts";
 
-import buildValidatorMacro from '../validator-macro'
-
-import { requiredObject } from '../../src/validators/requiredObject'
-
-const testMessage = 'This is required'
-const validator = requiredObject(testMessage)
-const requiredTest = buildValidatorMacro(validator, { _self: [ testMessage ] })
-
-test('fails on undefined', requiredTest, undefined, false)
-
-test('succeeds on empty array', requiredTest, [], true)
-test('succeeds on filled array', requiredTest, [ 0 ], true)
-
-test('succeeds on object', requiredTest, {}, true)
+const testMessage = "This is required";
+const validator = requiredObject(testMessage);
+const requiredTest = buildValidatorTest(validator, { _self: [testMessage] });
 
+// Deno.test("fails on undefined", () => {
+//   requiredTest(undefined, false);
+// });
```

Comment:
This should probably not be commented out?

Reply:
Changed